config/backend-config/backend-config.go

```diff
@@ -34,10 +36,21 @@ var (
 	curSourceJSONLock                sync.RWMutex
 	initialized                      bool
 	LastSync                         string
+
+	//DefaultBackendConfig will be initialized be Setup to either a WorkspaceConfig or MultiWorkspaceConfig.
+	DefaultBackendConfig BackendConfig
 )
 
 var Eb = new(utils.EventBus)
 
+const (
+	/*TopicBackendConfig topic provides updates on full backend config, via Subscribe function */
+	TopicBackendConfig = "backendConfig"
```

Comment:
Consider adding a new type for the topics. Compiler will not complain if string is used instead of the custom type. But I feel it will be more readable if we have the custom type. https://play.golang.org/p/u2i_YVTiyQ_Z

Reply:
Done!